docs/usage/filters.md

```diff
@@ -0,0 +1,195 @@
+# Filter knowledge
+
+In OpenCTI, you can filter data to target or view entities that have particular attributes.
+
+## Filters usages
+Filters are used in different locations in the platform.
+- in entities lists: to display only the entities matching the filters. If an export or a background task is generated, only the filtered data will be taken into account.
+- in investigations and knowledge graphs: to display only the entities matching the filters.
+- in workspaces: to create graphs with only the entities matching the filters.
+- in feeds, taxii collections, triggers, streams, playbooks, background tasks: to process only the data  or events matching the filters.
+
+There are two types of filters:
+- dynamic filters: they are not stored in the database, they enable to filter view in the UI. Examples: filters in entities list, investigations, knowledge graphs;
+- stored filters: They are attributes of an entity, they are stored as an attribute in the object. Examples: filters of workspaces, feeds, taxii collections, triggers, streams, playbooks.
+
+## Filters format (since 5.12)
+The filters format since OpenCTI 5.12 is of type FilterGroup (see below). This is the format to use when doing query via the API.
+This enables to do complex filters imbrication with different operators ('and', 'or').
+
+```
+FilterGroup: {
+    mode: 'and' | 'or',
+    filters: Filter[],
+    filterGroups: FilterGroup[],
+}
+
+Filter {
+    key: string,
+    values: string[],
+    operator: string,
+    mode: 'and' | 'or',
+}
+```
+
+In a filter group, the 'mode' can be equal to 'and' or 'or' and represent the operator between the different filters and groups in the 'filters' and 'filterGroups' arrays.
+The 'filters' and 'filterGroups' arrays are composed of objects of type Filter/FilterGroup.
+The base entity of a filter group object is a filter with 4 attributes:
+- a key, representing on what we want to do the filtering (exemple: 'objectLabel' to filter on labels, 'createdBy' to filter on the entity creation date, 'report_types', 'entity_type'... ).
+- an array of values, representing the values we are interested in.
+- an operator representing the operation we want to apply between the key and the values.
+- a mode ('and' or 'or') to apply between the values if there are several ones.
+
+### Operators
+The available operators are:
+- ```'eq'```  for 'equal',
+  
+
+    Exemple: ``entity_type = 'Report'``
+
+
+- ```'not_eq'``` for 'unequal',
+- ```'gt'/'gte'``` for 'greater than'/'greater than or equal',
+
+
+    Exemple: ``confidence > 50``
```

Comment:
I don't understand why there is a `gt` operator and a `>` in the example. is it to enlight that `>` is represented by `gt` in the filter code ?

Reply:
right that's a mistake. I'm going to fix this.